tap_github/schemas/events.json

```diff
@@ -1,296 +1,1055 @@
+                    },
+                    "tags_url": {
+                      "type": ["null", "string"]
+                    },
+                    "teams_url": {
+                      "type": ["null", "string"]
+                    },
+                    "trees_url": {
+                      "type": ["null", "string"]
+                    },
+                    "clone_url": {
+                      "type": ["null", "string"]
+                    },
+                    "mirror_url": {
+                      "type": ["null", "string"]
+                    },
+                    "hooks_url": {
+                      "type": ["null", "string"]
+                    },
+                    "svn_url": {
+                      "type": ["null", "string"]
+                    },
+                    "homepage": {
+                      "type": ["null", "string"]
+                    },
+                    "language": {
+                      "type": ["null", "string"]
+                    },
+                    "forks_count": {
+                      "type": ["null", "integer"]
+                    },
+                    "stargazers_count": {
+                      "type": ["null", "integer"]
+                    },
+                    "watchers_count": {
+                      "type": ["null", "integer"]
+                    },
+                    "size": {
+                      "type": ["null", "integer"]
+                    },
+                    "default_branch": {
+                      "type": ["null", "string"]
+                    },
+                    "open_issues_count": {
+                      "type": ["null", "integer"]
+                    },
+                    "is_template": {
+                      "type": ["null", "boolean"]
+                    },
+                    "topics": {
+                      "type": ["null", "array"],
+                      "items": {
+                        "type": ["null", "string"]
+                      }
+                    },
+                    "has_issues": {
+                      "type": ["null", "boolean"]
+                    },
+                    "has_projects": {
+                      "type": ["null", "boolean"]
+                    },
+                    "has_wiki": {
+                      "type": ["null", "boolean"]
+                    },
+                    "has_pages": {
+                      "type": ["null", "boolean"]
+                    },
+                    "has_downloads": {
+                      "type": ["null", "boolean"]
+                    },
+                    "archived": {
+                      "type": ["null", "boolean"]
+                    },
+                    "disabled": {
+                      "type": ["null", "boolean"]
+                    },
+                    "visibility": {
+                      "type": ["null", "string"]
+                    },
+                    "pushed_at": {
```

Comment:
date time format should be added for the fields "pushed_at", "created_at" and "updated_at"

Reply:
Actually in the API doc there are 4 fields named `pushed_at` two of which ar mentioned as pure string and other 2 are mentioned as date-time formats